cg/server/admin.py

```diff
@@ -392,10 +392,10 @@ class AnalysisView(BaseView):
     column_default_sort = ("created_at", True)
     column_editable_list = ["is_primary"]
     column_filters = ["pipeline", "pipeline_version", "is_primary"]
-    column_formatters = {"family": CaseView.view_family_link}
+    column_formatters = {"case": CaseView.view_family_link}
```

Comment:
```suggestion
    column_formatters = {"case": CaseView.view_case_link}
```
Just as a note; feel free to ignore for this PR. Looks like something that can be easily overlooked

Reply:
I'll make sure to go over this in a separate PR, I've tested the views currently and don't want to redo it 😅